gitlab/radiuss-jobs/lassen.yml

```diff
@@ -8,6 +8,7 @@
 clang_12_0_1_ibm_gcc_8_3_1_cuda_10_1_243:
   variables:
     SPEC: "${PROJECT_LASSEN_VARIANTS} +cuda %clang@=12.0.1.ibm.gcc.8.3.1 ^cuda@10.1.243+allow-unsupported-compilers ${PROJECT_LASSEN_DEPS}"
+    MODULE_LIST: "cuda/10.1.243"
```

Comment:
To make choosing jobs easier for projects, would it be better to only have jobs corresponding to system defaults for compilers and packages?  For example, CUDA 11.2.0 is the default now on LC blueos machines and there may be only one project, if any still using a CUDA 10.x version.

In other words, if you want to test anything other than system defaults I can add it to project file. If it's in here, I have to turn it off.

Reply:
The rationale for choosing those jobs was to list was projects need in production. If the needs have changed, we just need to change those spec accordingly.
This addition is just to load the appropriate module for the spec.

Reply:
I'll start an issue here to collect compilers and versions projects use in production. Hopefully, we can then agree on what we want in the shared specs.